daemon/logger/copier.go

```diff
@@ -69,3 +73,8 @@ func (c *Copier) copySrc(name string, src io.Reader) {
 func (c *Copier) Wait() {
 	c.copyJobs.Wait()
 }
+
+// Closed closed the copier
+func (c *Copier) Close() {
+	c.closed = true
```

Comment:
Hmm, go race detector will complain that you're reading and writing to the same variable at the same time. The runtime can even crash because of this.
I'd prefer to protect this var somehow or use a channel for that task.